gh2/csv.py

```diff
@@ -206,25 +214,23 @@ def set_headers(repo, labels=None):
         headers.extend('Label: ' + label.name for label in labels)
     return headers
 
+def get_all_labels(repositories, token):
+    labels = []
+    for repository in repositories:
+        repo_owner, repo_name = repository.split('/', 1)
+        repo = get_repo(repo_owner, repo_name, token)
+        repo_labels = get_repo_labels(repo)
+        for repo_label in repo_labels:
+           if repo_label.name not in [l.name for l in labels]:
```

Comment:
I don't think there is any need to keep the label objects, if you just use the names you can use a set which would allow you to replace this for-loop with
```python
all_labels.update(repo_labels)
```